draft-irtf-cfrg-hpke.md

```diff
@@ -1205,6 +1205,23 @@ interface; see {{hpke-export}} for more details.
 
 # API Considerations {#api-considerations}
 
+## Auxiliary Application Information
```

Comment:
Don't leave the lead-in empty.

This section documents considerations for interfaces to implementations of HPKE.  This includes recommendations that improve interoperability when HPKE is used in applications.

Reply:
In the text, “auxiliary _authenticated_ information” is used twice. We could use that in the title, too.

Reply:
👍 done